### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by hyper8432** [thatcherclough/AttendanceBot#6]
Hey, I just wanted to ask if this still works, and what if I have to choose my name from a drop-down list on the google form can the bot do that?? also I have no idea how to operate this bot, can you assist me, please?

**Comment by thatcherclough** [comment]
Yes, this is still working. To select a name from a drop down, you can inspect element of the drop down menu and see the options of the drop down. If you get the selector of the option, you can treat it like a button. The README explains how to use it for the most part. If you have any other questions let me know.

**Comment by hyper8432** [comment]
uhm im sorry, I dont understand anything on coding java, is there any video tutorial explanation about installing this attendancebot ?? like step by step

**Comment by hyper8432** [comment]
Should I give you the google form link to help me make the program works??

**Comment by hyper8432** [comment]
Oh yeah i forgot that my google form attendance also require me to Upload Photo, is it able to upload photo automatically tho?

**Comment by thatcherclough** [comment]
Unfortunately no.
